src/stomp_moveit_planner_plugin.cpp

```diff
@@ -48,8 +57,7 @@ class StompPlannerManager : public PlannerManager
       return nullptr;
     }
 
-    PlanningContextPtr planning_context =
-        std::make_shared<StompPlanningContext>("STOMP", req.group_name, param_listener_->get_params());
+    PlanningContextPtr planning_context = std::make_shared<StompPlanningContext>("STOMP", req.group_name, params_);
```

Comment:
Let's keep using `get_params()` to support changing parameters (https://github.com/PickNikRobotics/generate_parameter_library#dynamic-parameters).